iaac/terraform/common/ack-sagemaker-controller/main.tf

```diff
@@ -1,7 +1,12 @@
+data "github_repository_file" "profile_controller_policy" {
+  repository = "awslabs/kubeflow-manifests"
+  branch     = "main"
+  file       = "awsconfigs/infra_configs/iam_ack_oidc_sm_studio_policy.json"
+}
```

Comment:
This is the bit I'm guessing we would never upstream as we probably would have to find a better way to do this without having to reach out to github.

Reply:
I think having the separate .json file for the IAM policy isn't as necessary as they thought. The contents are pretty straightforward and only used in one other location. Maybe the right answer is simply hardcode the policy in the module like we do in our source. Thoughts?